README.md

```diff
@@ -1,2 +1,321 @@
+        },
+        ...
+    ],
+    users_in_progress: [],  // list of “in progress” users
+    experiments: [  // list of experiments definitions
+        {
+            experiment_name: <string>,
+            start_time: <Int32>,
+            end_time: <Int32>
+        },
+        ...
+    ],
+    course_id: <string>,  // course id
+    model_params: [  // list with models parameters
+        {
+            threshold: <double>,  // <double> is bson float type
+            pg: <double>,
+            ps: <double>,
+            pi: <double>,
+            pt: <double>,
+        },
+        ...
+    ],
+}
+```
+
+#### `Generic` collection stores all student - skills related data:
+
+
+```js
+{
+    _id: ObjectID(),
+    key: <string>,  // unique compound key, which contains <course_id>, <user_id>, and <skill_name>
+    val: {
+          threshold: <double>,
+          pg: <double>,
+          ps: <double>,
+          pi: <double>,
+          pt: <double>,
+    }
+}
+```
+
+### Course specific collections:
+
+#### `<course_id>_log` collects all logs about student's interactions
+
+Adaptive logging includes page load data and students responses on
+problems questions.
+
+##### Interaction log doc:
+
+```js
+{
+     _id: ObjectID(),
+     unix_s: <Int32>,
+     attempt: <Int32>,
+     student_id: <string>,
+     timestamp: <string>,
+     problem: {
+         skills: [],
+         pretest: false,
+         posttest: false,
+         tutor_url: <problem_url>,
+    },
+    type: "response",
+    correct: 0  // possible values 1 and 0 for correct and incorrect answers correspondingly
+}
+```
+
+##### Page load log doc:
+
+```js
+{
+     _id: ObjectID(),
+     unix_s: <Int32>,
+     student_id: <string>,
+     timestamp: <string>,
+     problem: {
+         skills: [],
+         pretest: false,
+         posttest: false,
+         tutor_url: <problem_url>,
+    },
+    type: "page_load"
+}
+```
+
+#### `<course_id>_problems` is a collection with the student's current status
+
+Status includes problems which were selected by Edx-Adapt for the
+student on every step through the adaptive course
+
+```js
+{
+     _id: ObjectID(),
+     current: {  // could be problem described dict or null object
+          problem: {
+          skills: [],
+          pretest: false,
+          posttest: false,
+          tutor_url: <problem_url>,
+     },
+     next: null,  // Possible values: dict with problem description, null.
+     student_id: <string>,
+     perm: false,  // (experimental attribute) boolean flag shows if student has permissions for fluent navigation.
+                   // Permission is changed after student start answering on post-assessment problems
+     nav: "adapt"  // Save initial student permission. Possible value “adapt” and “free”, “adapt” is default behavior
+                   // (perm == false), “free” (perm == true) student had permissions for free navigation when them had
+                   // started working
```

Comment:
"student had permissions for free navigation when them had started working"
What do you want to say here?